doc/README

```diff
@@ -9,17 +9,17 @@ Following features are supported:
     * ability to lock in-memory lookup index (hash table) to eliminate memory swap
     * readahead games with data and index blobs for maximum performance
     * multiple blob files support (tested with blob-file-as-block-device too)
-    * optional sha256 on-disk checksumming
+    * optional chunked crc32 on-disk checksumming
     * 2-stage write: prepare (which reserves the space) and commit
-    	(which calculates checksum and update in-memory and on-disk indexes).
-	    One can (re)write data using pwrite() in between without locks
+        (which calculates checksum and update in-memory and on-disk indexes).
+        One can (re)write data using pwrite() in between without locks
     * usual 1-stage write interface
     * flexible configuration of hash table size, flags, alignment
     * defragmentation tool: entries to be deleted are only marked as removed,
-    	eblob_check tool can actually remove those blocks from blob files
-	    by creating a copy and optionally rename it to original file
+        eblob_check tool can actually remove those blocks from blob files
```

Comment:
I just fixed tabulation. If there is no eblob_check tool, these lines should be updated.